src/Components/Button/Button.stories.jsx

```diff
@@ -0,0 +1,26 @@
+import { React } from 'react'
+import Button from './Button'
+
+export default {
+    title: 'Button component',
+    component: Button
+}
+
+const Template = (args) => {
+    const {
+        className, onClick, children, ariaLabel
+    } = args
+    return (
+        <div>
```

Comment:
In this case it doesn't matter but usually try not to leave unnecessary html tag.

Reply:
My Eslin auto formater converting like this, and I not save in other format. Try to change it?